ibm/conns/config.go

```diff
@@ -1809,10 +1802,18 @@ func (c *Config) ClientSession() (interface{}, error) {
 	session.apigatewayAPI = apigatewayAPI
 
 	// POWER SYSTEMS Service
-	ibmpisession, err := ibmpisession.New(sess.BluemixSession.Config.IAMAccessToken, c.Region, false, session.bmxUserDetails.UserAccount, c.Zone)
+	piURL := ContructEndpoint(c.Region, "power-iaas.cloud.ibm.com")
+	ibmPIOptions := &ibmpisession.IBMPIOptions{
+		Authenticator: authenticator,
+		Debug:         os.Getenv("TF_LOG") != "",
+		Region:        c.Region,
+		URL:           EnvFallBack([]string{"IBMCLOUD_PI_API_ENDPOINT"}, piURL),
+		UserAccount:   userConfig.UserAccount,
+		Zone:          c.Zone,
```

Comment:
Is there an option to pass custom header to SDK just like other services?

Reply:
No such option. SDK will set couple of headers for token and CRN automatically.